app/encrypt/aes.go

```diff
@@ -0,0 +1,51 @@
+// Package encrypt provides utilities to encrypt and decrypt data.
+package encrypt
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"io"
+)
+
+// AES256GCM encrypts the plaintext using AES-256-GCM with the provided key.
+// It returns the ciphertext with the nonce prepended.
```

Comment:
is prepending the nonce completely "standard"? It should probably be explain in the service doc

Reply:
Added the details in swagger doc. It's standard for implementations in `go` I think, but maybe not for everything. BTW if Michel has trouble with decrypting I can help, I made a few mistakes during the implementation so I know some potential mistakes.